logging/overlays/nerc-ocp-obs/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
+
+patches:
+  - path: externalsecrets/openshift-logging-lokistack-gateway-bearer-token_patch.yaml
```

Comment:
Let's remove this patch to logging/overlays/nerc-ocp-test because the LokiStack will create this secret.